### ISSUE TITLE
Spacing issue

### ISSUE THREAD

**Issue by ciops** [a62527776a/vue-floating-action-button#29]
Hey, there's a spacing issue on the first element when using "big" size. It also affects custom spacing options.

![image](https://user-images.githubusercontent.com/9364135/113730249-64b04580-9700-11eb-9028-f89e99c8dc4f.png)

**Comment by a62527776a** [comment]
Thanks for your feedback!

```
<vue-fab 
      size="big"
      :globalOptions="{spacing: 50,delay: 0.1}" // Please try to increase the line
/>
```

If it doesn't work, please send me a code snippet.

thank you!

**Comment by ciops** [comment]
I've created a new Vue project just to be sure there's no conflict with existing code, problem persists:

![image](https://user-images.githubusercontent.com/9364135/113864452-959d8280-97b3-11eb-8334-0bd1ac6483df.png)

Changing the spacing value doesn't affect the first element.

```
<template>
    <vue-fab 
        icon="menu" 
        size="big" 
        :scrollAutoHide="false" 
        :globalOptions="{ spacing: 50, delay: 0.05 }" 
        :mainBtnColor="mainBtnColor" 
        >
        <!-- :style="{ transform: 'translateX(10px)' }" -->
        <fab-item 
            v-for="(item, idx) in menu"
            :idx="idx"
            :key="idx"
            :title="item.title"
            :color="item.color"
            :icon="item.icon"
            @clickItem="navigateTo" 
            />
    </vue-fab>
</template>

<script>
export default {
    name: "FloatingNavigation",
    data () {
        return {
            menu: [
                {
                    idx: 0,
                    title: 'actionable',
                    icon: 'whatshot',
                    color: '#ff9900',
                    titleColor: '#fff',
                    titleBgColor: '#000',
                },
                {
                    idx: 1,
                    title: 'assets',
                    icon: 'savings',
                    color: '#729DCB'
                },
                {
                    idx: 2,
                    title: 'stats',
                    icon: 'analytics',
                    color: '#729DCB'
                },
                {
                    idx: 3,
                    title: 'profile',
                    icon: 'perm_identity',
                    color: '#729DCB'
                }
            ],
            mainBtnColor: '#729DCB'
        }
    },
    methods: {
        navigateTo: function (item) {
            return;
        },

    }
}
</script>
```

Also, `titleColor` and `titleBgColor` won't change the title color/background.

Awesome work on this project, love it!

**Comment by a62527776a** [comment]
thank you!
I missed this situation.
This problem is fixed in the latest @0.7.9 version!
You can check the latest situation in https://vue-fab.github.io/!

<img width="503" alt="企业微信截图_b6b76853-0677-4d1e-9a25-3592d76faea9" src="https://user-images.githubusercontent.com/13726832/113981351-2474d280-987a-11eb-9ce6-b3635cd771f3.png">

**Comment by ciops** [comment]
Works, thank you so much, really appreciated!

I want to suggest using the spacing attribute to define the space between elements (excluding their size). This way, the elements are evenly spaced out (including the first one) in any context. Something like this:

```
let itemSize;
if (this.$parent.size == 'normal') {
    itemSize = 32;
} else if (this.$parent.size == 'big') {
    itemSize = 38;
} else if (this.$parent.size == 'small') {
    itemSize = 26;
}
defaultY = -((this.idx+1) * itemSize + (this.idx+1) * this.$parent.globalOptions.spacing) + 'px';
```
